src/test/java/com/powsybl/openloadflow/network/impl/LfSwitchTest.java

```diff
@@ -68,7 +67,7 @@ void getterTest() {
 
     @Test
     void setterTest() {
-        lfSwitch.getPiModel().setX(LfBranch.LOW_IMPEDANCE_THRESHOLD); //FIXME
+        lfSwitch.getPiModel().setX(OpenLoadFlowParameters.load().getLowImpedanceThreshold());
```

Comment:
Can you add a test where you change the default value with `OpenLoadFlowParameters`? It will show you if all is working or not.